scripts/src/main/resources/scripts/functions

```diff
@@ -898,34 +898,19 @@ function doBackup() {
 # $3: backup path
 # $4-$N: contents of $1 ($1/* so all files in the folder of $1 after globbing)
 function doReplaceExtractedSkipSingleFolder() {
-  local source_dir="${1}"
-  local target_dir="${2}"
-  local backup_dir="${3}"
-  local root_dir="${1}"
+  local source_path="${1}"
+  local target_path="${2}"
+  local backup_path="${3}"
+  local root_path="${1}"
   shift 3
-  local files
-  local anz_files
-  local dir="${1}"
-  if [ "${#*}" = 1 ] && [ "${1%.app}" = "${1}" ] && [ -d "${1}" ]
+  local path="${1}"
```

Comment:
Also `path` is the same as `source_path` so one variable instead of 3 would be sufficient and easier to read and understand.

Reply:
done